.ci/azure/linux.yml

```diff
@@ -104,7 +105,14 @@ jobs:
     displayName: 'Build Lin'
 
   - script: ls -alR $(OPENVINO_REPO_DIR)/bin/
-    displayName: 'List files'
+    displayName: 'List bin files'
+
+  - script: cmake -DCMAKE_INSTALL_PREFIX=$(INSTALL_DIR) -P cmake_install.cmake
```

Comment:
I'm curious why we use `cmake_install.cmake` instead of `make install`. Could you please explain to me?

Reply:
it's a cross platform :)